mlflow/java/client/src/test/java/org/mlflow/tracking/MlflowClientTest.java

```diff
@@ -298,11 +303,12 @@ public void checkParamsAndMetrics() {
     assertParam(params, "max_depth", MAX_DEPTH);
 
     List<Metric> metrics = run.getData().getMetricsList();
-    Assert.assertEquals(metrics.size(), 4);
+    Assert.assertEquals(metrics.size(), 7);
```

Comment:
Can we also test that `metrics` contains `pos_inf` and `neg_inf` keys with the appropriate doubles?

Reply:
@tomasatdatabricks